packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -59,6 +59,98 @@ export const components = ({
   shadowRootElement?: HTMLDivElement;
 }): ThemeOptions["components"] => {
   return {
+    MuiAccordion: {
+      styleOverrides: {
+        root: () => ({
+          backgroundColor: odysseyTokens.HueNeutralWhite,
+          borderWidth: 1,
+          borderInlineStyle: "solid",
+          borderColor: odysseyTokens.HueNeutral100,
+          boxShadow: odysseyTokens.DepthMenu,
+
+          "&.noShadow": {
+            boxShadow: "none !important",
+          },
+
+          "&.Mui-disabled": {
+            backgroundColor: odysseyTokens.HueNeutralWhite,
+            color: odysseyTokens.TypographyColorDisabled,
+            cursor: "default",
+
+            "& .MuiAccordionSummary-content": {
+              color: odysseyTokens.TypographyColorDisabled,
+            },
+          },
+
+          "&::before": {
+            backgroundColor: odysseyTokens.BorderColorDisplay,
+            opacity: "1 !important",
+          },
```

Comment:
Can we remove these `!important` declarations?

We can hack it like `&::before::before` though I think.